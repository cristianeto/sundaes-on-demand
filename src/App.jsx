import { Container } from 'react-bootstrap';
import { OrderDeatilsProvider } from './contexts/OrderDetails';
import OrderEntry from './pages/entry/OrderEntry';

function App() {
  return (
    <Container>
      <OrderDeatilsProvider>
        {/** Sumary page and entry page need provider */}
        <OrderEntry />
      </OrderDeatilsProvider>
      {/**Confirmation page does not need a provider*/}
    </Container>
  );
}

export default App;
