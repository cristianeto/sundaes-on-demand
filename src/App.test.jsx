import { render } from '@testing-library/react';
import App from './App';

test('<App />', () => {
  const { baseElement } = render(<App />);

  expect(baseElement).toMatchSnapshot();
  expect(baseElement).toBeInTheDocument();
});
