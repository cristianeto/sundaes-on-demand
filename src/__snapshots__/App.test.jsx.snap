// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> 1`] = `
<body>
  <div>
    <form
      class=""
    >
      <div>
        <div
          class="form-check"
        >
          <input
            class="form-check-input"
            id="terms-and-conditions"
            type="checkbox"
          />
          <label
            class="form-check-label"
            for="terms-and-conditions"
            title=""
          >
            <span>
              I agree with
              <span
                style="color: blue;"
              >
                Terms and Conditions
              </span>
            </span>
          </label>
        </div>
      </div>
      <button
        class="btn btn-primary"
        disabled=""
        type="submit"
      >
        Confirm order
      </button>
    </form>
  </div>
</body>
`;

exports[`renders learn react link 1`] = `
<body>
  <div>
    <form
      class=""
    >
      <div>
        <div
          class="form-check"
        >
          <input
            class="form-check-input"
            id="terms-and-conditions"
            type="checkbox"
          />
          <label
            class="form-check-label"
            for="terms-and-conditions"
            title=""
          >
            <span>
              I agree with
              <span
                style="color: blue;"
              >
                Terms and Conditions
              </span>
            </span>
          </label>
        </div>
      </div>
      <button
        class="btn btn-primary"
        disabled=""
        type="submit"
      >
        Confirm order
      </button>
    </form>
  </div>
</body>
`;
